cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/ExternalNetworkInjectionConversion.java

```diff
@@ -50,6 +51,15 @@ public void convert() {
         convertReactiveLimits(g);
 
         context.regulatingControlMapping().forGenerators().add(g.getId(), p);
+
+        addSpecificProperties(g, p);
     }
 
+    private static void addSpecificProperties(Generator generator, PropertyBag p) {
+        generator.setProperty(Conversion.PROPERTY_CGMES_ORIGINAL_CLASS, "ExternalNetworkInjection");
```

Comment:
Should `ExternalNetworkInjection` also be a constant (like `CgmesNames.EQUIVALENT_INJECTION`)?